README.md

```diff
@@ -39,7 +39,7 @@
     <tbody align="center">
         <tr>
             <td >2.3.*</td>
-            <td rowspan=6><a href="https://github.com/dotnet/spark/releases/tag/v0.12.1">v0.12.1</a></td>
+            <td rowspan=10><a href="https://github.com/dotnet/spark/releases/tag/v1.0.0">v1.0.0</a></td>
```

Comment:
Maybe we should redo the table as follow moving forward?

<table>
    <thead>
        <tr>
            <th>Apache Spark</th>
            <th>.NET for Apache Spark</th>
        </tr>
    </thead>
    <tbody align="center">
        <tr>
            <td >2.3</td>
            <td rowspan=3><a href="https://github.com/dotnet/spark/releases/tag/v1.0.0">v1.0.0</a></td>
        </tr>
        <tr>
            <td>2.4*</td>
        </tr>
        <tr>
            <td>3.0</td>
        </tr>
    </tbody>
</table>
* 2.4.2 is <a href="https://github.com/dotnet/spark/issues/60">not supported</a>.

Thoughts? cc @rapoth

Reply:
or is it better to make it explicit?

Reply:
I think if there are no more plans to release 2.4.x then we can use this compact table instead.

Reply:
Yep, this new one looks now concise.